sys-cluster/torque/files/torque-6.1.3-port-args.patch

```diff
@@ -0,0 +1,46 @@
+diff -up torque/contrib/init.d/pbs_mom.in.port-args torque/contrib/init.d/pbs_mom.in
```

Comment:
Link, but is this even relevant for us? Do we use their init script?

Reply:
No, we don't